artifacts/manifests/cronjob.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: publisher
+  namespace: publishing-bot
```

Comment:
why is setting the namespace necessary?

Reply:
It's better to regroup all the resources related to publishing-bot in the same namespace instead of using the `ðefault` namespace on the new cluster.

Reply:
i thought that if you don't specify the namespace `kubectl apply` will just use the "current" namespace.

Reply:
@mfojtik What's the issue by specifying the namespace ?

Reply:
this code is generic. There are other consumers than just kube. Please don't change this. Namespace is intensionally missing.